File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        unordered_map<int, int>mpp;
        int n = numbers.size();

        for(int i = 0; i < n; i++){
            if(mpp.find(target-numbers[i]) != mpp.end()){
                return {(mpp[target-numbers[i]])+1, i+1};
            }
            mpp[numbers[i]] = i;
        }
        return {};
    }
};

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        mpp = {}
        
        for i in range(len(numbers)):
            if target-numbers[i] in mpp:
                return [mpp[target-numbers[i]]+1, i+1]
            else:
                mpp[numbers[i]] = i
        
        return  []
                